### ISSUE TITLE
Output degradation - Only serviceAccount is printed and not the attack vector (serviceAccount-role-roleBinding) 

### ISSUE THREAD

**Issue by clintmod** [armosec/kubescape#295]

**Comment by clintmod** [comment]
v1.0.135 works correctly… so I'll change the title from v1.0.137 to v1.0.136 where it was introduced

**Comment by clintmod** [comment]
After looking at this a bit more I think I am only seeing kind `ServiceAccount` show up in the excluded namespaces.

**Comment by slashben** [comment]
We are checking

**Comment by dwertent** [comment]
Thank you @clintmod 
I tested the `exclude-namespaces` on latest and previous versions of kubescape and I don't see a difference in the behavior. However, we have some degradation in the display and that is probably confusing you.

I will explain - When using `exclude-namespaces`, the kubescape request from the k8s-api is similar to `kubectl get --field-selector=metadata.namespace!=kube-system` and for this reason the resources in the namespace are excluded from scanning - they are not pulled at all from the k8s-api. 
When you have a `serviceAccount` attached to a `ClusterRoleBinding` as a subset, you will see this `serviceAccount` as failed in the results because the `ClusterRoleBinding` is not filtered out by the k8s-api.

This is why we recommend on using the [kubescape exceptions](https://github.com/armosec/kubescape/blob/master/examples/exceptions/README.md), with the exceptions kubescape filters the namespace of the resource without any API limitations

I hope I answered your question :)
